script/tool/lib/src/common/core.dart

```diff
@@ -30,6 +30,10 @@ const String platformWindows = 'windows';
 /// Key for enable experiment.
 const String kEnableExperiment = 'enable-experiment';
 
+/// Target platforms supported by Flutter.
+// ignore: public_member_api_docs
+enum FlutterPlatform { android, ios, linux, macos, web, windows }
```

Comment:
I added this since I was adding a utility method to `RepositoryPackage` to get the platform directories of a package. Currently we use string constants everywhere for platforms which I've been wanting to move to enums, and this was a good opportunity to introduce it. (I didn't convert anything else to it yet.)